src/07-projects/projects.apib

```diff
@@ -205,6 +205,12 @@ Update a project.
                 + done
                 + cancelled
         + starts_on: `2016-02-04` (string)
+        + customer (object) - Null will unlink the customer from the project
```

Comment:
Q: Reading the extra information given, did you mean to make customer nullable?
```apib
        + customer (object, nullable) - Null will unlink the customer from the project
```

Reply:
I did that, but then I saw this does nothing in the generated documentation, it does not mention it is nullable 🤷

Reply:
That is weird indeed 🤔 I thought this was visible somewhere, it should be at least in the Json Schema itself.